extensions/iceworks-material-import/README.en.md

```diff
@@ -9,10 +9,17 @@ Add materials to the application in a visual way.
 
 ## Usage
 
-![import-materials](https://img.alicdn.com/tfs/TB1wvG5aAcx_u4jSZFlXXXnUFXa-1279-904.gif)
+### Activate Extension
 
-1. Right click the location where you want to insert the material and select「iceworks: import material」, or click iceworks in the status bar of VSCode and select 「iceworks: import material」
-2. Click the block or the component and the code will be added to your code
+![demo](https://user-images.githubusercontent.com/56879942/87401390-a63a1700-c5ec-11ea-91a6-f7d81ac5fd77.gif)
+
+1. Arouse vscode command palette  through `⇧⌘P`.
```

Comment:
Maybe some user who use the windows don't konw what the `⇧⌘P` stands for? For example me. I suggest that you can add Arouse vscode command palette  through `⇧⌘P` or `Ctrl + Shift + P`. What do you think about it?

Reply:
OK, I have changed this in all docs.